Makefile

```diff
@@ -110,6 +111,12 @@ fmt: install-tools ## Formats all go files
 	$(GO) run .github/tools/matrixchecker/main.go
 	./build/docker-go-version.sh Dockerfile ./suppression-backup-service/Dockerfile
 
+.PHONY: sec
```

Comment:
We can merge it with the existing `sec:` command.
https://github.com/rudderlabs/rudder-server/blob/master/Makefile#L130-L134